integration/mediation-tests/tests-transport/src/test/resources/testng.xml

```diff
@@ -21,12 +21,12 @@
         </classes>
     </test>
 
-    <!--<test name="ViewPopRedirectTests" preserve-order="true" verbose="2">
+    -<test name="ViewPopRedirectTests" preserve-order="true" verbose="2">
```

Comment:
What is purpose of "-" at the beginning?

Reply:
fixed